doc/release/1.5.0-notes.rst

```diff
@@ -39,39 +60,247 @@ New features
+footprint reduction between 25 % and 60 % (see gh-11069).
+
+`scipy.stats` functions that accept a seed for reproducible calculations using 
+random number generation (e.g. random variates from distributions) can now use 
+the modern ``np.random.Generator`` as well as the legacy 
+``np.random.RandomState`` as a seed.
+
+The ``axis`` parameter was added to `scipy.stats.rankdata`. This allows slices 
+of an array along the given axis to be ranked independently.
+
+The ``axis`` parameter was added to `scipy.stats.f_oneway`, allowing it compute 
+multiple one-way ANOVA tests for data stored in n-dimensional arrays. Some
+performance improvements to ``f_oneway`` as well.
+
+The PDF and CDF methods for ``stats.geninvgauss`` are now significantly faster 
+as  the numerical integration to calculate the CDF uses a Cython based 
+``LowLevelCallable``.
+
+Moments of the normal distribution (`scipy.stats.norm`) are now calculated using 
+analytical formulas instead of numerical integration for greater speed and 
+accuracy
+
+Moments and entropy trapezoidal distribution (`scipy.stats.trapz`) are now 
+calculated using analytical formulas instead of numerical integration for 
+greater speed and accuracy
+
+Methods of the truncated normal distribution (`scipy.stats.truncnorm`), 
+especially ``_rvs``, are significantly faster after a complete rewrite.
+
+The `fit` method of the Laplace distribution,  `scipy.stats.laplace`, now uses 
+the analytical formulas for the maximum likelihood estimates of the parameters.
+
+Generation of random variates is now thread safe for all SciPy distributions. 
+3rd-party distributions may need to modify the signature of the ``_rvs()`` 
+method to conform to ``_rvs(self, ..., size=None, random_state=None)``. (A 
+one-time VisibleDeprecationWarning is emitted when using non-conformant 
+distributions.)
+
+The Kolmogorov-Smirnov two-sided test statistic distribution 
+(`scipy.stats.kstwo`) was added. Calculates the distribution of the K-S 
+two-sided statistic ``D_n`` for a sample of size n, using a mixture of exact 
+and asymptotic algorithms.
+
+The new function ``median_abs_deviation`` replaces the deprecated 
+``median_absolute_deviation``.
+
+The ``wilcoxon`` function now computes the p-value for Wilcoxon's signed rank 
+test using the exact distribution for inputs up to length 25.  The function has 
+a new ``mode`` parameter to specify how the p-value is to be computed.  The 
+default is ``"auto"``, which uses the exact distribution for inputs up to length 
+25 and the normal approximation for larger inputs.
+
+Added a new Cython-based implementation to evaluate guassian kernel estimates,
+which should improve the performance of ``gaussian_kde``
+
+The ``winsorize`` function now has a ``nan_policy`` argument for refined
+handling of ``nan`` input values.
+
+The ``binned_statistic_dd`` function with ``statistic="std"`` performance was
+improved by ~4x.
+
+``scipy.stats.kstest(rvs, cdf,...)`` now handles both one-sample and 
```

Comment:
perhaps put the 'kstest' higher up next to the 'kstwo' item as they both relate to K-S statistics?